src/main/docs/guide/pubsub/pullConsumer/subscriberProperties.adoc

```diff
@@ -1,4 +1,4 @@
-Pub/Sub allows each link:https://googleapis.dev/java/google-cloud-pubsub/latest/com/google/cloud/pubsub/v1/Subscriber.html[Subscriber] to have its own configuration for things such as executors or flow control settings.
+Pub/Sub allows each Pull link:https://googleapis.dev/java/google-cloud-pubsub/latest/com/google/cloud/pubsub/v1/Subscriber.html[Subscriber] to have its own configuration for things such as executors or flow control settings.
```

Comment:
is this `Pull` addition intentional?

Reply:
Yes, because these settings do not apply to Push subscriptions and I wanted to make that clear.